docs/content/SelectMenu.md

```diff
@@ -40,6 +40,42 @@ SelectMenu components get `COMMON` system props. Read our [System Props](/system
 | :- | :- | :-: | :- |
 | initialTab | String | | If using the `SelectMenu.Tabs` component, you can use this prop to change the tab shown on open. By default, the first tab will be used.
 
+## SelectMenu.MenuContext
+SelectMenu.MenuContext is a [context object](https://reactjs.org/docs/context.html#reactcreatecontext) that exposes some helpful state values to be used via [`React.useContext`](https://reactjs.org/docs/hooks-reference.html#usecontext) in consuming applications.  SelectMenu.MenuContext can only be used in components that are already wrapped in a `SelectMenu` as `SelectMenu` contains the [context provider](https://reactjs.org/docs/context.html#contextprovider).
+
+### Values and functions available on MenuContext
```

Comment:
Functions are values so this seemed redundant to me. I don't feel strongly about this though.

```suggestion
### Values available on MenuContext
```